src/app/view/components/_common/dot-action-button/dot-action-button.component.ts

```diff
@@ -8,15 +9,46 @@ import { MenuItem } from 'primeng/primeng';
  * @class ActionButtonComponent
  */
 @Component({
-    encapsulation: ViewEncapsulation.None,
     selector: 'dot-action-button',
     styleUrls: ['./dot-action-button.component.scss'],
     templateUrl: 'dot-action-button.component.html'
 })
 
-export class DotActionButtonComponent {
-    @Input() model?: MenuItem[];
-    @Input() command?: ($event) => void;
+export class DotActionButtonComponent implements OnInit {
+    @ViewChild('menu') menu: Menu;
+
+    @Input() disabled: boolean;
+    @Input() icon: string;
     @Input() label: string;
-    @Input() flat: boolean;
+    @Input() model: MenuItem[];
+    @Input() selected: boolean;
+
+    @Output() onClick: EventEmitter<any> = new EventEmitter();
```

Comment:
How primeng does it now: https://github.com/primefaces/primeng/blob/master/src/app/components/splitbutton/splitbutton.ts#L59